TikokulympicBeta/Views/Profile/ProfileView.swift

```diff
@@ -48,9 +47,7 @@ struct ProfileView: View {
                             VStack(alignment: .leading, spacing: 15) {
                                 HStack {
                                     AuthProfileImage()
-                                    Text("名前: \(userName)")
-                                        .font(.title3)
-                                    Image(systemName: "pencil")
+                                    NameEdit()
```

Comment:
```suggestion
        HStack {
            if isEditing {
                TextField("名前を入力", text: $userName)
                    .font(.title3)
                    .textFieldStyle(RoundedBorderTextFieldStyle())
            } else {
                Text("名前: \(userName)")
                    .font(.title3)
            }

            Button(action: {
                isEditing.toggle()
            }) {
                Image(systemName: "pencil")
            }
        }
        .padding()
```
今回はこんな感じでProfileViewにべたがきでいいかな〜